content/en/docs/started/workstation/minikube-linux.md

```diff
@@ -93,10 +96,14 @@ Follow these steps to install Kubeflow:
   page](https://github.com/kubeflow/kfctl/releases/tag/{{% kf-latest-version %}}).
 
 1. Unpack the tar ball:
-      ```
-      tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
-      ```
-1. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the kfctl binary each time you run it.
+   
+   ```SHELL
+   tar -xvf kfctl_{{% kf-latest-version %}}_<platform>.tar.gz
+   ```
+1. Run the commands below to set up and deploy Kubeflow. One of them
+   includes an optional command to add the binary kfctl to your
+   path. If you don’t add the binary to your path, you must use the full path
+   to the kfctl binary each time you run it.
 
     ```
     # The following command is optional, to make kfctl binary easier to use.
```

Comment:
```suggestion
    # The following command is optional making the kfctl binary easier to use.
```

Also Line 110:
```
+ # Replace `{PATH_FOR_KFTCL} with your path where kfctl should be extracted to.
- export PATH=$PATH:<path to where kfctl was unpacked>
+ export PATH=$PATH:{PATH_FOR_KFTCL}
```

and further below it:

```
- export KF_NAME=<your choice of name for the Kubeflow deployment>
+ export KF_NAME={YOUR_KF_DEPLOYMENT_NAME}
```

```
- # Set the path to the base directory where you want to store one or more 
+ # Set the path to the base directory - change {YOUR_PATH_TO_BASE_DIR} -
- # Kubeflow deployments. For example, /opt/.
+ # to store your Kubeflow deployment(s). For example: `/opt/`.
- export BASE_DIR=<path to a base directory>
+ export BASE_DIR={YOUR_PATH_TO_BASE_DIR}
```

The use of the dot/full stop is not clear otherwise.)

WDYT?

I think it's also good to to use `<>` instead of `{}`.